### PR TITLE
BD-3333. Use FIO native amount calculations

### PR DESCRIPTION
**Pull request by trukhilio**
- Use native fio amount on wallets calculations

### CODE REVIEW

File: client/src/components/FioNameRenew/FioNameRenewContainer.tsx [hunk 1]
```diff
@@ -65,7 +70,9 @@ const FioNameRenewContainer: React.FC<ContainerProps> = props => {
   };
 
   const hasLowBalance =
-    currentWallet && feePrice && currentWallet.balance < feePrice.costFio;
+    currentWallet &&
+    feePrice &&
+    walletBalancesTotal.nativeFio < feePrice.costFio;
```
Comment (andreyvEze) [inline client/src/components/FioNameRenew/FioNameRenewContainer.tsx:75]: are you sure `costFio` is native?

File: client/src/components/FioNameTransfer/components/FioNameTransferForm/FioNameTransferForm.tsx [hunk 2]
```diff
@@ -33,10 +35,12 @@ export const TransferForm = (props: FormProps) => {
   const [validating, setValidating] = useState<boolean>(false);
   const [valid, setValid] = useState<boolean>(true);
 
+  const { total: walletBalancesTotal } = useWalletBalances(publicKey);
+
   const { costFio, costUsdc } = feePrice;
   const fioNameLabel = fioNameLabels[fioNameType];
   const hasLowBalance =
-    currentWallet && feePrice && currentWallet.balance < feePrice.costFio;
+    publicKey && feePrice && walletBalancesTotal.nativeFio < feePrice.costFio;
```
Comment (andreyvEze) [inline client/src/components/FioNameTransfer/components/FioNameTransferForm/FioNameTransferForm.tsx:43]: are you sure `costFio` is native?

File: client/src/pages/FioAddressAddBundlesPage/FioAddressAddBundlesPage.tsx [hunk 3]
```diff
@@ -56,8 +56,12 @@ const FioAddressAddBundlesPage: React.FC<ContainerProps &
     getFee();
   }, []);
 
+  const { total: walletBalancesTotal } = useWalletBalances(
+    currentWallet.publicKey,
+  );
+
   const hasLowBalance =
-    currentWallet && feePrice && currentWallet.balance < feePrice.costFio;
+    feePrice && walletBalancesTotal.nativeFio < feePrice.costFio;
```
Comment (andreyvEze) [inline client/src/pages/FioAddressAddBundlesPage/FioAddressAddBundlesPage.tsx:64]: are you sure costFio is native?

File: client/src/pages/FioDomainStatusChangePage/FioDomainStatusChangePage.tsx [hunk 4]
```diff
@@ -82,7 +85,7 @@ const FioDomainStatusChangePage: React.FC<ContainerProps> = props => {
   };
 
   const hasLowBalance =
-    fioWallet && feePrice && fioWallet.balance < feePrice.costFio;
+    feePrice && walletBalancesTotal?.nativeFio < feePrice.costFio;
```
Comment (andreyvEze) [inline client/src/pages/FioDomainStatusChangePage/FioDomainStatusChangePage.tsx:88]: check costFIo

File: client/src/components/FioNameRenew/FioNameRenewContainer.tsx (hunk 1 above)
Comment (trukhilio) [reply client/src/components/FioNameRenew/FioNameRenewContainer.tsx:75]: it would native in next commit

**Review by andreyvEze — APPROVED** [review]

**Pull request merged**